CtrlVAF/CtrlVAF.AdditionalAssembly/TestHandler.cs

```diff
@@ -1,22 +1,24 @@
-﻿using CtrlVAF.Commands.Commands;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace CtrlVAF.Commands.Handlers
-{
-    public class TestConfiguration
-    {
-        public int id = 0;
-    }
-
-    public class TestHandler : ICommandHandler<BeforeCheckInChangesCommand<TestConfiguration>>
-    {
-        public void Handle(BeforeCheckInChangesCommand<TestConfiguration> command)
-        {
-            command.Env.CurrentUserID = command.Configuration.id;
-        }
-    }
-}
+﻿using CtrlVAF.Commands.Commands;
```

Comment:
Is this testhandler actually used anywhere?
If not, lets just get rid of the AdditionalAssembly project for now :))

Reply:
It's used in one of your tests. It was in CtrlVAF.Core at first, I just moved it to a different assembly altogether to keep CtrVAF.Core cleaner.